pkg/generator/generator.go

```diff
@@ -285,8 +285,17 @@ func ComputeFingerprint(args ...interface{}) string {
 
 func AddAssetChecks(cfg *config.Config, a config.Target, l log.Logger) error {
 	checks := []config.Check{}
+	var policy config.Policy
 	for ref, ch := range cfg.CheckTypes {
-		if stringInSlice(a.AssetType, ch.Assets) && filterChecktype(ch.Name, cfg.Conf.IncludeR, cfg.Conf.ExcludeR) {
+		if cfg.Conf.Policy != "" {
+			policy = GetPolicy(cfg)
+		}
+		if stringInSlice(a.AssetType, ch.Assets) && filterChecktype(ch.Name, cfg.Conf.IncludeR, cfg.Conf.ExcludeR, policy) {
```

Comment:
With this approach if a policycheck is referencing a check type that doesn't exists it will remain unnoticed for the user.
I think it could make more sense to iterate the policy.checktypes and find the related checktype. 
```
for pc in policy.Checks
   ch = findchecktype(pc.name)
   if not found 
      log.alert(not found)
      continue
   
   add check
```

It could be more efficient and easy to follow. and perhaps move to another function and not mix with the filterCheckType regex functionality

Reply:
Thanks Jesús, I agree. Now when a policy is set, the tool iterates through the checks specified in the policy [905815b](https://github.com/adevinta/vulcan-local/pull/5/commits/905815b70a5d863b3ea0b1717260ccdf8c432c46).